wiki/Contests/VMC/1/en.md

```diff
@@ -0,0 +1,84 @@
+| --: | :-- |
+| Announcement | 2023-05-01 |
+| Submission | 2023-05-01 to 2023-05-31 |
+| Judging | 2023-06-01 to 2023-06-24 |
+| Results | 2023-07-01 |
+
+## Staff and Judges
+
+The Vocaloid Mapping Contest is hosted and organised by the [Vocaloid Appreciation Team (VAT)](https://discord.gg/EruvDHaReY), specifically:
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ::{ flag=US }:: [-White](https://osu.ppy.sh/users/16276548) |
+| Staff | ::{ flag=IT }:: [Kuroise](https://osu.ppy.sh/users/14547127), ::{ flag=US }:: [Local Hero](https://osu.ppy.sh/users/16134122), ::{ flag=SG }:: [Ixcors](https://osu.ppy.sh/users/11194087) |
+| Judge | ::{ flag=US }:: [Noffy](https://osu.ppy.sh/users/1541323), ::{ flag=US }:: [-White](https://osu.ppy.sh/users/16276548), ::{ flag=CA }:: [Shanipika](https://osu.ppy.sh/users/6336729), ::{ flag=CN }:: [Mafumafu](https://osu.ppy.sh/users/3076909), ::{ flag=SV }::[Natsu](https://osu.ppy.sh/users/1953876), ::{ flag=US }:: [tutuhaha](https://osu.ppy.sh/users/546991) |
+| Designer | ::{ flag=VE }:: [Mithia](https://osu.ppy.sh/users/9748438) |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| 1st place | 6 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points), *VMC Winner* profile badge, assistance in the ranking process |
+| 2nd place | 2 months of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |
+| 3rd place | 1 month of osu!supporter, [*Elite Mapper* contest points](/wiki/Contests/Contest_points) |
+
+All participating mappers received constructive feedback from some of the most well-known vocal synth mappers in the community.
+
+## Links
+
+- [Announcement news post](https://osu.ppy.sh/home/news/2023-05-01-vocaloid-mapping-contest)
+- [Contest listing](https://osu.ppy.sh/community/contests/177)
+- [Discord server](https://discord.gg/EruvDHaReY)
+
+## Ruleset
+
+- **Submissions must be for the osu! ruleset only.** Comparing beatmaps of different modes in a judging scenario is unfortunately not realistic.
+- **Submissions must use one of the [selected songs](#songs)**
+- **Submissions must be in `.osz` format.**
+- **Beatmaps should abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be disqualified.
+- **Submissions must not be publicly revealed until after the results are announced.**
+- **Excluding metadata, submissions must not include anything that could reveal your identity.** Metadata will be anonymised before submissions are relayed to judges.
+- **Collaborations are not allowed**. This is a solo contest.
+- **Submissions must not include storyboards or videos.**
+- **Do not edit any settings in the .osu that would not be possible through the editor itself.** You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.
+- **You must submit only one difficulty.** Only one entry is allowed.
```

Comment:
```suggestion
- **Your submission must have only one difficulty.**
```
if you're using the osu contest listing, it's implied that only one submission is allowed anyway